package.json

```diff
@@ -6,14 +6,21 @@
     "type": "git",
     "url": "git+https://github.com/staltz/callbag-share.git"
   },
-  "main": "index.js",
+  "main": "index.cjs.js",
+  "module": "index.js",
   "scripts": {
-    "test": "tape test.js"
```

Comment:
Where does `$npm_package_module` come from? Some new npm? rollup?

Reply:
https://docs.npmjs.com/misc/scripts#packagejson-vars

Not sure when it was added, but it it's available for quite some time. I can inline contents of those, but it nicely guards against accidental spelling mistakes and such.

Reply:
Nice! Seems like it's supported since October 2017 or even before that, so should cover npm v5.